cli/pcluster/utils.py

```diff
@@ -296,7 +296,8 @@ def get_supported_os(scheduler):
     :param scheduler: the scheduler for which we want to know the supported os
     :return: an array of strings of the supported OSes
     """
-    oses = ["alinux", "alinux2"]
+    # NOTE: alinux2 is first because it should be the default in `pcluster configure`
+    oses = ["alinux2", "alinux"]
     if scheduler != "awsbatch":
```

Comment:
There's got to be a better way to handle this. Would it make sense to define an `easyconfig_default` in mappings.py where easyconfig could get the default value from?

Reply:
Yes I'd define a mapping like this, but in the `easyconfig.py` module.
Let's use a structure similar to the one used in the funciton `get_default_values_for_required_clsuter_section_params` you defined .

But for now we can keep the code as is, it's up to you.

Reply:
Addressed in the last commit I pushed.